### PR TITLE
HW-14.2

### PR DESCRIPTION
**Pull request by serg-ksv**

### CODE REVIEW

File: src/main/java/mate/academy/internetshop/Main.java [hunk 1]
```diff
@@ -1,35 +1,80 @@
...
             System.out.println(product.toString());
         }
+
+        User user1 = new User("user1", "user1@mail.com", "somePass1");
+        User user2 = new User("user2", "user2@mail.com", "somePass2");
+        User user3 = new User("user3", "user3@mail.com", "somePass3");
+        userService.create(user1);
+        userService.create(user2);
+        userService.create(user3);
+        for (User user : userService.getAll()) {
```
Comment (boroda4436) [inline src/main/java/mate/academy/internetshop/Main.java]: better to use stream API for each method

File: src/main/java/mate/academy/internetshop/Main.java [hunk 2]
```diff
@@ -1,35 +1,80 @@
...
+        User getUser = userService.get(3L);
+        System.out.println("Get user3:\n" + getUser);
+        System.out.println("Update user1:\n" + userService.update(userService.get(1L)));
+        System.out.println("Delete user2:" + userService.delete(2L));
+        for (User user : userService.getAll()) {
+            System.out.println(user.toString());
+        }
+
+        final ShoppingCart user1Cart = shoppingCartService.getByUserId(user1.getId());
+        shoppingCartService.addProduct(shoppingCartService.getByUserId(user1.getId()), product1);
```
Comment (boroda4436) [inline src/main/java/mate/academy/internetshop/Main.java]:
```suggestion
        shoppingCartService.addProduct(user1Cart), product1);
```
you have already declare the variable. Use it

File: src/main/java/mate/academy/internetshop/Main.java [hunk 3]
```diff
@@ -1,35 +1,80 @@
...
+                shoppingCartService.getAllProducts(shoppingCartService.getByUserId(user1.getId()));
+        System.out.println("User1's products:");
+        for (Product product : user1Products) {
+            System.out.println(product);
+        }
+        shoppingCartService.deleteProduct(user1Cart, product1);
+        System.out.println("User1's products:\n" + user1Cart.getProducts());
+
+        Order user1Order = orderService.completeOrder(shoppingCartService.getAllProducts(
+                shoppingCartService.getByUserId(user1.getId())), user1);
```
Comment (boroda4436) [inline src/main/java/mate/academy/internetshop/Main.java]:
```suggestion
         user1Cart)), user1);
```

**Review by boroda4436 — CHANGES_REQUESTED** [review]
Where are services and their implementations?

File: src/main/java/mate/academy/internetshop/Main.java (hunk 1 above)
Comment (serg-ksv) [reply src/main/java/mate/academy/internetshop/Main.java]: done

File: src/main/java/mate/academy/internetshop/Main.java (hunk 2 above)
Comment (serg-ksv) [reply src/main/java/mate/academy/internetshop/Main.java]: done

File: src/main/java/mate/academy/internetshop/Main.java (hunk 3 above)
Comment (serg-ksv) [reply src/main/java/mate/academy/internetshop/Main.java]: done

**Comment by serg-ksv** [comment]
> Where are services and their implementations?

I was in a hurry and didn't notice that the files were not added.

**Review by boroda4436 — APPROVED** [review]

**Pull request merged**